cloudslang-content-verifier/src/main/java/io/cloudslang/lang/tools/build/tester/SlangTestRunner.java

```diff
@@ -37,21 +43,30 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeoutException;
+
+import static java.util.concurrent.TimeUnit.MINUTES;
+
 
-/**
- * Created by stoneo on 3/15/2015.
- */
 @Component
 public class SlangTestRunner {
 
     private final static String PROJECT_PATH_TOKEN = "${project_path}";
+    public static final long MAX_TIME_PER_TESTCASE_IN_MINUTES = 10;
```

Comment:
maybe it would be nice to set this value from the builder's configuration folder (cslang.properties)